app/controllers/api/receipts_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Api::ReceiptsController < ApplicationController
+  respond_to :json
+  def show
+  end
+
+  def create
+    receipt = Receipt.new(receipt_params)
+    if receipt.save!
+      receipt.request.open = false
```

Comment:
Also need to change availability of work

Reply:
flash notifications: request marked as closed, work marked as rented/sold would also be nice

Reply:
not super sure how receipts work - if a receipt is created that means the request is closed?